frontend/README.md

```diff
@@ -1,35 +1,58 @@
-* You need to answer questions and complete the code exercise
-    * Code exercise result must be on Stackblitz or Github
-    * Answers can be in Russian or English language
-* The result of the entire assignment must be on Google Docs or Github
-
-## Assignment
-#### Block 1: Common
-    1. What are primitives? How can developers use them?
-    2. What does keyword `this` mean?
-    3. What are callbacks?
-    4. What is a function declaration and how are its types different?
-    5. Compare spread and rest operators.
-#### [Code exercise](https://stackblitz.com/edit/js-v99jvs)
-
-#### Block 2: Async
-    1. JavaScript, is it synchronous or not?
-    2. What is the key principle of Call Stack?
-    3. What does AJAX mean?
-    4. Promise hell. How would you solve it?
-    5. How would you handle exceptions in JS? List two options.
-    6. Which framework would you choose for a new project? Explain why.
-#### [Code exercise](https://stackblitz.com/edit/js-llr1ac-async-task-chvwqk)
-
-#### Block 3: TypeScript
-    1. Is it possible to check types in runtime? If so, how?
-    2. What is the difference between private and protected fields?
-    3. How do interfaces help in development?
-    4. How would you pass arguments into a Class?
-#### [Code exercise](https://stackblitz.com/edit/typescript-2xfiqk)
+The repo contains the server part of a simple spend-tracking application written using ME*N stack.
 
+ME*N stack technologies
+- MongoDB - document database
+- Express(.js) - Node.js web framework
+- Any client-side JavaScript framework - React, Angular, Vue, Vanilla JS
+- Node(.js) - the premier JavaScript web server
+
+Express and Node make up the middle (application) tier. Express.js is a server-side web framework, and Node.js the popular and powerful JavaScript server platform.
+
+You need to write the client side of spend-tracking application using any JS framework / UI kit / libs you like.
+
+Application will need to display, add new, update, remove expenses and track statistics by month and category.
+
+## Getting Started
+
+Clone the repo
+```
+git clone git@github.com:provectus/internship.git
+cd /internship/frontend/exercise
+```
+
+Pre-fill the database with the generated data
+```
+docker-compose up mongo-seed
+```
+
+Start the api server
+```
+docker-compose up api
+```
+
+Check the api doc generated by the server side
+
+```
+http://localhost:5000/api-docs
+```
+![Api](./swagger.png)
+## Acceptance criteria
+
+### Difficulty Level : Easy
+
+- Minimalistic UI implemented to display / create / update / delete expences
+- UI contains chart to display statistics by month and category
+- Added comprehensive documentation to launch the application
+
+### Difficulty Level : Medium
+
+- The app is written using TypeScript
+- The forms contain validations
+
+### Difficulty Level : High
+
+- The client application is wrapped in a Docker container
+- The compose file updated, added section to run the whole application with one command
```

Comment:
`The docker-compose file is updated and can be used to run the whole app with one command`